pkg/lca/seedgenerator.go

```diff
@@ -230,6 +232,12 @@ func (builder *SeedGeneratorBuilder) WithRecertImage(
 		return builder
 	}
 
+	if recertImage == "" {
+		glog.V(100).Infof("The name of the recertImage is empty")
+
+		builder.errorMsg = "recertImage 'name' cannot be empty"
```

Comment:
shouldn't U return builder in this clause? otherwise `builder.Definition.Spec.RecertImage` is updated with an empty value

Reply:
Updated. Thanks!